src/python/aqueduct_executor/operators/systemmetric_executor/main.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import base64
+import traceback
+import sys
+
+from aqueduct_executor.operators.utils import enums, utils
+from aqueduct_executor.operators.systemmetric_executor import spec
+from aqueduct_executor.operators.utils.storage.parse import parse_storage
+
+
+def run(spec: spec.SystemMetricSpec) -> None:
+    """
+    Executes a system metric operator by storing the requested system metrics value in the output content path.
+    """
+    storage = parse_storage(spec.storage_config)
+    system_metadata = utils.read_system_metadata(storage, spec.input_metadata_paths)
+    utils.write_artifact(
+        storage,
+        spec.output_content_paths[0],
+        spec.output_metadata_paths[0],
+        float(system_metadata[0]["SystemMetadata"][spec.metricname]),
```

Comment:
Doesn't this mean that we expect `input_metadata_paths` to have length 1? We should enforce this assumption on the spec itself then right? Why does the spec allow for multiple input and output paths?

Reply:
yup, I added enforcement on spec creation